### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by top-notch-coder**
First commit

### CODE REVIEW

**Comment by Talented0Dev** [comment]
new1

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-## App Built on top of this Starter project [IDURAR ERP/CRM](https://github.com/idurar/erp-crm)
```
Comment (Talented0Dev) [inline README.md:1]: Typing error

Comment (top-notch-coder) [inline README.md:1]: sorry Typing error

Comment (top-notch-coder) [reply README.md:1]: right

Comment (Talented0Dev) [reply README.md:1]: 👍👍👍

Comment (top-notch-coder) [reply README.md:1]: 😊😊😊

**Pull request merged**